social_core/tests/test_utils.py

```diff
@@ -199,7 +200,7 @@ def _backend(self, session_kwargs=None):
 class GetKeyAndSecretBasicAuthTest(unittest.TestCase):
     def setUp(self) -> None:
         self.backend = BaseAuth(strategy=Mock())
-        self.backend.setting = Mock(
+        self.backend.setting: Any = Mock(
```

Comment:
pyright fails here:
  /home/runner/work/social-core/social-core/social_core/tests/test_utils.py:203:31 - error: Type annotation not supported for this statement (reportInvalidTypeForm)